src/modules/NWHeader.tsx

```diff
@@ -1,25 +1,28 @@
-import { useAppDispatch, useAppSelector } from '../stuff/hooks';
-import { GameModes } from '../stuff/Shared';
-import { selectGameMode, setGameMode } from '../stuff/slices/GameSlice';
-import { selectIsMobileSized } from '../stuff/slices/StylingSlice';
+import {useAppDispatch, useAppSelector} from '../stuff/hooks';
 import { selectWgEmojified } from '../stuff/slices/WordGridSlice';
 
 import './NWHeader.css';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faShareSquare} from '@fortawesome/free-solid-svg-icons';
+import {setHappyToastMessage} from '../stuff/slices/GameSlice';
 
 export const NWHeader = () => {
   const wgEmoji = useAppSelector(selectWgEmojified);
 
-  // TODO: Make this actually work and put it on the bar.
+  const dispatch = useAppDispatch();
+
   const onCopyEmojiClicked = () => {
-    alert(wgEmoji);
+    navigator.clipboard.writeText(wgEmoji).then(() => {
+      dispatch(setHappyToastMessage('Game results copied!'));
+    });
   };
   const alertEmojiButton = (
-    <button onClick={onCopyEmojiClicked}>copy emoji</button>
+    <FontAwesomeIcon onClick={onCopyEmojiClicked} icon={faShareSquare}></FontAwesomeIcon>
   );
 
   return (
     <div className='NWHeader'>
-      <p>Networds</p>
+      <p>Networds <span className={'Clickable'}>{alertEmojiButton}</span></p>
```

Comment:
The reason for the `<p>` tag was actually just to put the text in something so I could lazily use `space-between` to force text to the left and buttons to the right, so I'd actually put the button outside the `<p>`. But then I also insist on using `<div>` for everything, so maybe my design decisions should be taken with a grain of salt.

Also TIL git comments will parse html tags.